recipe-server/client/control_new/components/data/QueryExtensionListingColumns.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { loadExtensionListingColumns } from 'control_new/state/extensions/actions';
+
+
+@connect(
+  null,
+  dispatch => (bindActionCreators({
```

Comment:
We sort of do this differently in certain components - in some we just pass an object of functions (`{ loadExtensionListingColumns }`) and other times we use `bindActionCreators`.

@rehandalal Is there any reason we should use one syntax over the other? If they're the same, I'd suggest just going with the object for brevity.